src/mlagility/cli/report.py

```diff
@@ -162,22 +178,47 @@ def summary_spreadsheet(args) -> None:
             build_name = state.config.build_name
 
             # Load MLAgility stats from the YAML file
-            mlagility_stats = filesystem.get_stats(cache_dir, build_name)
+            # FIXME: Use mlagility_stats for hash and parameters
+            # https://github.com/groq/mlagility/issues/174
+            # mlagility_stats = filesystem.get_stats(cache_dir, build_name)
```

Comment:
Suggest sequencing this PR after #188 so that you don't have to add these FIXMEs

Reply:
Ack!